### PR TITLE
[IRRC-74] Create form for creating a new Teacher

### PR DESCRIPTION
**Pull request by dzldebby**

### CODE REVIEW

File: frontend/src/pages/NewTeacher.vue [hunk 1]
```diff
@@ -0,0 +1,413 @@
...
+    showAddSecondLanguage() {
+      this.$buefy.dialog.prompt({
+        message: `Add new language`,
+        inputAttrs: {
+          placeholder: 'e.g. Italian',
+          maxlength: 255,
+        },
+        confirmText: 'Add',
+        onConfirm: (value) => {
+          this.$refs.secondlanguagevalue.setSelected(value)
```
Comment (thchia) [inline frontend/src/pages/NewTeacher.vue]: Your onConfirm here and the onConfirm for the first language look quite different? If the first one is working I suggest following that, or perhaps referring to the documentation to see which is the correct way to set the value after it's been entered.

**Review by aelbore — APPROVED** [review]

**Comment by dzldebby** [comment]
Changes made only to newteacher.vue, please ignore all other files

**Review by lauratsui — APPROVED** [review]
LGTM, will add in the relevant changes for #75

**Comment by dzldebby** [comment]
@thchia This PR has gotten 2 approvals and can be merged, but is it okay if I just resolve the conflicts in students.js by removing those lines that are coming from new-teacher?

**Comment by thchia** [comment]
@dzldebby yeah I think that looks right, please go ahead.

**Pull request merged**